src/ProtobufServer/ecdar_requests/send_query.rs

```diff
@@ -153,6 +153,7 @@ fn convert_ecdar_result(query_result: &QueryResult) -> Option<ProtobufResult> {
                     reason: "".to_string(),
                     relation: vec![],
                     state: None,
+                    action: "".to_string(),
```

Comment:
Will this always be empty? If so, I think another implementation would be beneficial. If this will be changed in the future, I think a comment should be added

Reply:
I see that it could be more clear, In these cases there will never be an action to add, we could use an empty string by just using String::from(""), else we will have to make the action optinal which we would prefer not to as it should not be optinal in some cases.

Reply:
Fair. Could you add a comment somewhere describing why this is the case for these? Otherwise, LGTM

Reply:
Yes, greate idea, have added it at the first case, I could not find a better place.